crates/chain/src/canonical_iter.rs

```diff
@@ -214,36 +213,39 @@ impl<A: Clone> CanonicalReason<A> {
     }
 
     /// Constructs a [`CanonicalReason`] from a `last_seen` value.
-    pub fn from_last_seen(last_seen: LastSeenIn) -> Self {
-        Self::LastSeen {
-            last_seen,
+    pub fn from_observed_in(observed_in: ObservedIn) -> Self {
+        Self::ObservedIn {
+            observed_in,
             descendant: None,
         }
     }
 
     /// Contruct a new [`CanonicalReason`] from the original which is transitive to `descendant`.
     ///
-    /// This signals that either the [`LastSeenIn`] or [`Anchor`] value belongs to the transaction's
+    /// This signals that either the [`ObservedIn`] or [`Anchor`] value belongs to the transaction's
     /// descendant, but is transitively relevant.
     pub fn to_transitive(&self, descendant: Txid) -> Self {
         match self {
             CanonicalReason::Anchor { anchor, .. } => Self::Anchor {
                 anchor: anchor.clone(),
                 descendant: Some(descendant),
             },
-            CanonicalReason::LastSeen { last_seen, .. } => Self::LastSeen {
-                last_seen: *last_seen,
+            CanonicalReason::ObservedIn {
+                observed_in: last_seen,
+                ..
+            } => Self::ObservedIn {
+                observed_in: *last_seen,
```

Comment:
Is it intentional to keep using `last_seen` here instead of all `observed_in`?

Reply:
Fixed